virtual-classroom/src/main/java/ro/utcn/sd/mid/assign1/virtualclassroom/entity/SOUser.java

```diff
@@ -0,0 +1,48 @@
+package ro.utcn.sd.mid.assign1.virtualclassroom.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Data
+@Entity(name = "sousers")
+@NoArgsConstructor @AllArgsConstructor
+public class SOUser implements IDEntity, NameEntity{
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    private String sOUsername;
+    private String sOPassword;
+    //private Boolean banned;
```

Comment:
Having commented code is usually not a good sign :) That is why we use git --> if you delete code from a previous commit, you can still find it on git by inspecting the file's history.